src/pages/governance/governance.module.scss

```diff
@@ -0,0 +1,123 @@
+@mixin description-text {
+  color: var(--ifm-color-emphasis-700);
+  letter-spacing: 0.01em;
+}
+
+@mixin dark-mode-invert {
+  [data-theme='dark'] & {
+    filter: invert(1);
+  }
+}
+
+.header {
+  margin: 0 auto;
+  max-width: 40rem;
+  text-align: center;
+
+  p {
+    @include description-text;
+  }
+}
+
+.cardWrapper {
+  margin: 1rem 0;
+}
+
+.card {
+  max-width: 30rem;
+  padding: 1rem;
+  margin: 0 auto;
+}
+
+.cardTitle {
+  font-size: 1.2rem;
+  font-weight: 500;
+  margin-bottom: 0.2em;
+}
+
+.cardSubtitle {
+  font-size: 1em;
+  font-weight: 500;
+  margin-bottom: 0.1rem;
+}
+
+.cardLink {
+  display: inline-block;
+  vertical-align: middle;
+  opacity: 1;
+  &:hover {
+    opacity: 0.5;
+  }
+  img {
+    height: 1em;
+    margin-left: 0.2em;
+    @include dark-mode-invert;
+  }
+}
+
+.cardIcon {
+  text-align: center;
+  color: var(--ifm-color-white);
+  line-height: var(--ifm-avatar-photo-size-sm);
+  // one-off colors
+  background: linear-gradient(to top, #dd5e89, #f7bb97);
+}
+
+.cardDescription {
+  display: block;
+  font-size: 0.9em;
+  min-height: 8em;
```

Comment:
```suggestion
  min-height: 2em;
```

I think we can reduce the height and still have it look OK. What do you think?

| 8em | 2em |
| -- | -- |
| ![Card with description min-height 8em](https://user-images.githubusercontent.com/606594/139905646-caf80016-622f-46bd-b01c-38539eb1a994.png) | ![Card with description min-height 2em](https://user-images.githubusercontent.com/606594/139905603-b793bef9-319a-4154-88e0-a24efe9c7b24.png) |

Reply:
The 8em height is to accommodate the longer copy in the Security WG README. It's about twice as long as the other ones, but I wanted all cards to remain the same height.